src/middlewared/middlewared/plugins/enclosure_/nvme2.py

```diff
@@ -35,12 +39,13 @@ def fake_nvme_enclosure(model, num_of_nvme_slots, mapped):
         device = mapped.get(slot, None)
         if device is not None:
             status = 'OK'
-            value_raw = '0x1000000'
+            value_raw = 16777216
         else:
-            status = 'Not Installed'
-            value_raw = '0x0500000'
+            status = 'Not installed'
+            value_raw = 83886080
```

Comment:
The old values were actually bugged in 2 ways:
1. they were a string and so were never being converted to their integer value
2. they were the wrong hex value

This value is what should be returned from sg3_utils api when we're talking to proper SES. Since this is NVMe (which uses PCIe), we "fake" the value. I can go back to using the proper hex value.